v4l2/v4l2_codecbase.h

```diff
@@ -101,7 +103,9 @@ class V4l2CodecBase {
     };
     virtual int32_t usePixmap(uint32_t bufferIndex, Pixmap pixmap) {return 0;};
     #endif
+    #if __ENABLE_EGL__
     virtual int32_t useEglImage(EGLDisplay eglDisplay, EGLContext eglContext, uint32_t buffer_index, void* egl_image) {return 0;};
+    #endif
```

Comment:
Nevermind the above comment.  EGLDisplay, EGLContext are only defined with ENABLE_EGL so this is ok